properties.py

```diff
@@ -264,6 +264,16 @@ def update_type(self, context):
         name="AOV name",
         description="Name of the built in AOV")
 
+    denoise_aov = BoolProperty(
+        name="Format AOV for denoising",
+        description="If checked this pass will be export as an individual file and properly formatted for use by the denoise utility",
+        default=False)
+
+    exclude_from_multi = BoolProperty(
```

Comment:
To that end, maybe as a long term project I'll try to implement a section in the output panel where a user can define one or more multilayer files and then assign passes to them......  Definitely above my skill set at the moment so it will take me some time to get it done.